physionet-django/user/views.py

```diff
@@ -573,12 +573,16 @@ def edit_credentialing(request):
 
 
 @login_required
-def user_credential_applications(request):
+def user_credential_applications(request, user=None):
     """
     All the credential applications made by a user
     """
+    if user:
+        request_user = User.objects.filter(username=user)[0]
```

Comment:
Instead of using a filter and indexing the first item, it seems cleaner to use `get` (we would expect usernames to be unique). [get_object_or_404](https://docs.djangoproject.com/en/3.1/topics/http/shortcuts/#get-object-or-404) might be even better.

Reply:
Nice catch!